projects/packages/my-jetpack/_inc/components/evaluation-recommendations/index.tsx

```diff
@@ -25,6 +68,21 @@ const EvaluationRecommendations: FC = () => {
 	const menuRedoTitle = __( 'Redo', 'jetpack-my-jetpack' );
 	const menuDismissTitle = __( 'Dismiss', 'jetpack-my-jetpack' );
 
+	useEffect( () => {
+		const container = containerRef.current;
+
+		if ( container ) {
+			container.addEventListener( 'scroll', checkScrollPosition );
+			checkScrollPosition();
+		}
+
+		return () => {
+			if ( container ) {
+				container.removeEventListener( 'scroll', checkScrollPosition );
+			}
+		};
+	}, [ checkScrollPosition ] );
```

Comment:
```suggestion
	useEffect( () => {
		const container = containerRef.current;

		if ( container ) {
			container.addEventListener( 'scroll', checkScrollPosition );
			checkScrollPosition();
			
			return () => {
				container.removeEventListener( 'scroll', checkScrollPosition );
		        };
		}
	}, [ checkScrollPosition ] );
```

Would this work the same way?